src/pages/Whitepaper/whitepaperContent.tsx

```diff
@@ -5,8 +5,8 @@ const htmlContent = `<!DOCTYPE html>
     <!-- Required meta tags -->
```

Comment:
I just noticed whitepaperContent has html body and head tags, they shouldn't be in there.

Reply:
Good spot. Updated below.